### ISSUE TITLE
`FURB113`: False positive

### ISSUE THREAD

**Issue by ofek** [astral-sh/ruff#8640]
```
Use `output.extend(...)` instead of repeatedly calling `output.append()`
```

```python
ordered = [identifier for identifiers in ordering_data.values() for identifier in reversed(identifiers)]
output = [
    'from __future__ import annotations',
    '',
    '# fmt: off',
    'ORDERED_DISTRIBUTIONS: tuple[str, ...] = (',
]
output.extend(f'    {identifier!r},' for identifier in ordered)
output.append(')')  # error here
```

**Comment by dhruvmanila** [comment]
I'm unable to reproduce this with the provided code snippet. Can you please provide the Ruff version you're running against?

**Comment by ofek** [comment]
`0.1.4`

**Comment by dhruvmanila** [comment]
Thanks, I'm unable to reproduce this with the following command:
```console
$ pipx run "ruff==0.1.4" check --select=FURB113 --no-cache --isolated src/FURB113.py --preview
```

Neither when using `--select=ALL`. Where are you running Ruff from? Command-line, editor, or as a pre-commit hook? Which command did you use to run Ruff if from the command-line?

**Comment by charliermarsh** [comment]
Seems like there's some missing context -- I don't see it in the playground: https://play.ruff.rs/94de3bbb-5f0f-47b0-9cfa-47587fac4d4b

**Comment by ofek** [comment]
```
git clone https://github.com/pypa/hatch
cd hatch
```

remove the ignore comment

https://github.com/pypa/hatch/blob/1e7f8ff29a5ca264c7a8531de70171f7184b1501/scripts/update_distributions.py#L66

then run

```
ruff check --isolated --preview --select FURB113 scripts/update_distributions.py
```

**Comment by dhruvmanila** [comment]
Thanks for the context!

I think it's correct as there's an `append` call right after that. So, 
```python
    output = [
        'from __future__ import annotations',
        '',
        '# fmt: off',
        'ORDERED_DISTRIBUTIONS: tuple[str, ...] = (',
    ]
    output.extend(f'    {identifier!r},' for identifier in ordered)
    output.append(')')  # noqa: FURB113
	# `FURB113` is highlighting to merge the above and below `append` calls
    output.append('DISTRIBUTIONS: dict[str, dict[tuple[str, ...], str]] = {')
```

**Comment by ofek** [comment]
I'm sorry 🤦‍♂️ my brain is fried from updating the entire code base this weekend

**Comment by dhruvmanila** [comment]
> I'm sorry 🤦‍♂️ my brain is fried from updating the entire code base this weekend

Haha, no worries. Happy to help :)
